tests/php/sync/test_class.jetpack-sync-integration.php

```diff
@@ -2,6 +2,52 @@
 
 class WP_Test_Jetpack_Sync_Integration extends WP_Test_Jetpack_Sync_Base {
 
+	function test_get_sync_status() {
```

Comment:
Should we create a new file for `Jetpack_Sync_Actions` tests specifically? This doesn't seem like an integration test anyway.

Reply:
This is what I was asking about on Slack. I was looking for an elaboration on this and an appropriate location.

Reply:
Done it in https://github.com/Automattic/jetpack/pull/12209/commits/2df11c975c4793895cfcec576d5929e7ec687332